tooling/helium-server/worker/ssr.js

```diff
@@ -46,14 +46,30 @@ async function handleSsr(url, authToken = null, userAndAppearanceToken = null) {
     sha256,
     authToken
   );
-  const { httpResponse } = pageContext;
+
+  const { httpResponse, redirectTo } = pageContext;
 
   if (!httpResponse) {
     return null;
   } else {
-    const { statusCode, body } = httpResponse;
+    let { statusCode, body } = httpResponse;
     const headers = assembleHeaders(pageContext);
 
+    if (!statusCode) {
+      //set default if code not provided
+      statusCode = 302;
+    }
+
+    if (redirectTo) {
```

Comment:
@AlexONeillTI Think we'll want to move the check for `redirectTo` before the check for `httpResponse`. Looking at `vite-plugin-ssr`'s example, looks like there's the possibility we could get a `redirectTo` back even without an `httpResponse`, and as it's currently written, this order would miss the redirect.